.github/workflows/actions.yml

```diff
@@ -304,13 +304,182 @@ jobs:
+            cmake-generator: 'MinGW Makefiles'
+            chocolatey-options: '' # '--forcex86' for 32-bit build
+            use-qtwebengine: 'OFF' # might need to be turned off for MinGW
+            ableton-link: 'OFF' # might need to be turned off for MinGW
+            artifact-suffix: 'win64-mingw' # set if needed - will trigger artifact upload
+
+    name: Windows ${{ matrix.job-name }}
+    env:
+      BUILD_PATH: ${{ github.workspace }}/build
+      INSTALL_PATH: ${{ github.workspace }}/build/Install
+      LIBS_DOWNLOAD_PATH: ${{ github.workspace }}/../3rd-party
+      ASIO_URL: 'http://www.steinberg.net/sdk_downloads/asiosdk2.3.zip'
+      ASIO_FOLDER_NAME: ASIOSDK2.3
+      VCVARS_SCRIPT: ${{ matrix.vcvars-script }}
+      VCVARS_SCRIPT_PATH: 'C:/Program Files (x86)/Microsoft Visual Studio/${{ matrix.msvc-year }}/Enterprise/VC/Auxiliary/Build/${{ matrix.vcvars-script }}'
+      ARTIFACT_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.artifact-suffix }}'
+      INSTALLER_FILE: 'SuperCollider-${{ needs.lint.outputs.sc-version }}-${{ matrix.installer-suffix }}'
+      CREATE_INSTALLER: ${{ matrix.create-installer }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: prepare timestamp for cache
+        id: current-date
+        run: |
+          echo "::set-output name=stamp::$(date '+%Y-%m-%d')"
+          echo "::set-output name=week::$(date '+%V')"
+      - name: cache qt
+        id: cache-qt
+        uses: actions/cache@v1
+        with:
+          path: ../Qt
+          key: ${{ runner.os }}-v1-${{ matrix.os-version }}-${{ matrix.qt-version }}-qt${{ matrix.qt-arch }}
+      - name: install qt using aqtinstall
+        uses: jurplel/install-qt-action@v2
+        with:
+          modules: 'qtwebengine'
+          version: ${{ matrix.qt-version }}
+          arch: ${{ matrix.qt-arch }}
+          cached: ${{ steps.cache-qt.outputs.cache-hit }}
+      - name: install libsndfile
+        run: choco install libsndfile ${{ matrix.chocolatey-options }}
+      - name: download fftw
+        shell: bash
+        env:
+          FFTW_PATH: ${{ env.LIBS_DOWNLOAD_PATH }}/fftw
+        run: |
+          mkdir -p $FFTW_PATH && cd $FFTW_PATH
+          curl -L ${{ matrix.fftw-url }} -o fftw.zip
+          7z x fftw.zip -y
+          # add env var for subsequent steps
+          echo 'FFTW_PATH<<EOF' >> $GITHUB_ENV
+          echo $FFTW_PATH >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: create fftw msvc library
+        if: env.VCVARS_SCRIPT
+        shell: cmd
+        env:
+          FFTW_ARCH: ${{ matrix.fftw-arch }}
+        run: |
+          call "%VCVARS_SCRIPT_PATH%"
+          cd "%FFTW_PATH%"
+          lib.exe /machine:%FFTW_ARCH% /def:libfftw3f-3.def
+      - name: move fftw
+        shell: bash
+        run: |
+          mv $FFTW_PATH "C:/Program Files/fftw"
+          echo "Done installing fftw"
```

Comment:
i think you can remove this echo since the feedback in the actions dashboard is pretty clear